src/paper/views/paper_views.py

```diff
@@ -402,6 +403,17 @@ def censor(self, request, pk=None):
 
         return Response("Paper was deleted.", status=200)
 
+    @action(
+        detail=True,
+        methods=["post"],
+        permission_classes=[],
+    )
+    def update_biorxiv_tweet_count(self, request, pk=None):
+        paper = self.get_object()
```

Comment:
Can we add a check here to ignore updating the tweet count if it has been updated recently?

Reply:
Yes, we will also no longer be calling this endpoint